tests/test_submissions.py

```diff
@@ -9,7 +9,7 @@
 from evalai.submissions import submission
```

Comment:
This should not be a part of this PR.

Reply:
It's because API_HOST_URL doesn't exist in challenges, because I've moved it in the PR.

Reply:
okay!